server/pyproject.toml

```diff
@@ -41,7 +41,6 @@ h11 = ">=0.13,<0.15"
 # Can use marshmallow or the toasted flavour as you like ;-)
 # marshmallow = "2.14.0"
 toastedmarshmallow = "0.2.6" # updating causes consistency problem in test
-trio = "0.22.2"              # see https://github.com/python-trio/trio-typing/issues/85
```

Comment:
Should we also remove `trio_typing` (and its corresponding `mypy` plugin below), `quart-trio` and `hypothesis-trio`?

Reply:
yes, it's under way ^^